torch_xla/csrc/xla_sharding_util.cpp

```diff
@@ -382,4 +383,55 @@ std::vector<at::Tensor> ShardingUtil::ShardTensor(
   return shards;
 }
 
+void ShardingUtil::PrepareOutputShardingPropagation(
+    std::vector<XLATensorPtr>* tensors, absl::Span<const size_t> indices,
+    ComputationPtr computation,
+    std::vector<torch::lazy::BackendDataPtr>* data_placeholders,
```

Comment:
why pointer instead of reference?

Reply:
No particular or a strong reason, just wanted to be consistent with other inputs `std::vector<XLATensorPtr>* tensors` .

Reply:
We can clean up this later. It seems weird to use raw pointers...